src/gepetto_viewer_rerun/client.py

```diff
@@ -0,0 +1,349 @@
+from dataclasses import dataclass
+import logging
+import numpy as np
+from math import tau
+from enum import Enum
+from typing import List, Union
+import rerun as rr
+
+from .entity  import (
+    Entity,
+    MeshFromPath
+)
+from .scene import Scene
+
+logger = logging.getLogger(__name__)
+
+
+class Archetype(Enum):
+    ARROWS3D = 0
+    BoxeS3D = 1
+    CAPSULES3D = 2
+    LINESTRIPS3D = 3
+    MESH3D = 4
+    MESH_FROM_PATH = 5
+    POINTS3D = 6
+    
+
+class Client:
+    def __init__(self):
+        self.gui = Gui()
+
+@dataclass
```

Comment:
```suggestion
```

Reply:
Why removing @dataclass here ? 
Gui defines multiple attributes.

Reply:
(I edited your comment, because dataclass is a github user, and they received a notification when you mentioned them like that)

`Gui` is not a class that is only here to own data. A dataclass only provide shortcuts to define `__init__` and `__repr__` methods for classes that only (or almost only) hold attributes. That `Gui` holds some attributes, but is mostly intended to run code.

Reply:
(Woops, didn't know about that :sweat_smile:)
I understand, thank you